apps/ledger-live-desktop/src/renderer/modals/UpdateFirmwareModal/steps/03-step-confirmation.tsx

```diff
@@ -56,51 +51,27 @@ const StepConfirmation = ({ error, deviceModelId }: StepProps) => {
         iconSize={24}
       />
       <Title mt={9}>
-        {t("manager.modal.successTitle", { deviceName: device ? device.productName : "" })}
+        {appsToBeReinstalled
+          ? t("manager.modal.successTitleApps", { deviceName: device ? device.productName : "" })
```

Comment:
you can do something like `const deviceName = device ? device.productName : ""` and use it 3 times :)